code/datums/abnormality/_ego_datum/teth.dm

```diff
@@ -330,3 +330,12 @@
 /datum/ego_datum/armor/faelantern
 	item_path = /obj/item/clothing/suit/armor/ego_gear/teth/faelantern
 	cost = 20
+
+//Dealer of the Damned - Luck of the Draw
+/datum/ego_datum/weapon/luckdraw
+	item_path = /obj/item/gun/ego_gun/luckdraw
+	cost = 20
+
+/datum/ego_datum/armor/luckdraw
+	item_path = /obj/item/clothing/suit/armor/ego_gear/teth/luckdraw
+	cost = 20
```

Comment:
Dont forget to leave a extra line of blank code at the end of files. Its a weird quirk of byond code that a extra blank line is needed in each file.